knora/excel2xml.py

```diff
@@ -669,7 +672,9 @@ def make_decimal_prop(
 
     # check value type
     for val in values:
-        if not re.search(r"^\d+\.\d+$", str(val.value).strip()):
+        try:
+            float(val.value)
+        except ValueError:
```

Comment:
Float validation and Int Validation: Instead of inventing my own validation, I let Python do the work for me, and I accept everything that Python accepts.